path_config

```diff
@@ -16,6 +16,11 @@ path /status StatusHandler {
     root .;
 }
 
+path /reverse_proxy ReverseProxyHandler {
+    remote_host localhost;
+    remote_port 1234;
```

Comment:
Yeah, that's what we thought as well, but that's what the TA concluded for now. But again, she said we should expect to see some more details on the spec, so things are subject to change I guess.